src/nvim/option_defs.h

```diff
@@ -493,9 +493,9 @@ EXTERN long p_mat;              /* 'matchtime' */
 EXTERN long p_mco;              /* 'maxcombine' */
 EXTERN long p_mfd;              /* 'maxfuncdepth' */
 EXTERN long p_mmd;              /* 'maxmapdepth' */
-EXTERN long p_mm;               /* 'maxmem' */
+EXTERN long p_mm;               /* 'maxmem' (ignored in Neovim) */
 EXTERN long p_mmp;              /* 'maxmempattern' */
-EXTERN long p_mmt;              /* 'maxmemtot' */
+EXTERN long p_mmt;              /* 'maxmemtot' (ignored in Neovim) */
```

Comment:
These are all documented in vim_diff.txt. I don't think we need to start doing it here too (otherwise there are many other comments that need to be updated).